.travis.yml

```diff
@@ -2,12 +2,7 @@ dist: xenial
 sudo: false
 
 rvm:
-  - "2.1"
-  - "2.2"
-  - "2.3"
-  - "2.4"
-  - "2.5"
-  - "2.6"
+  - "2.7"
```

Comment:
The [Toolbelt still uses 2.3](https://github.com/aptible/omnibus-aptible-toolbelt/blob/4adbab6dcf932cb0ffa7f29fc812a3a65bad029a/config/projects/aptible-toolbelt.rb#L37), so we can't stop testing it.
```
$ /opt/aptible-toolbelt/embedded/bin/ruby --version
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin19]
```

Reply:
sadface, I'll see what I can do to get BigNumber behaving

Reply:
finally figured it out - it was Bundler that was the biggest pain